docs/openapi/components/schemas/common/EntrySummary.yml

```diff
@@ -0,0 +1,396 @@
+    $linkedData:
+      term: importDate
+      '@id': https://schema.org/Date
+  billOfLadingNumber:
+    title: Bill of Lading Number
+    description: Bill of Lading or Waybill number.
+    type: string
+    $linkedData:
+      term: billOfLadingNumber
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncl1153/#Bill_of_lading_number
+  manufacturerId:
+    title: Manufacturer Identifier
+    description: This block is provided to accommodate the manufacturer/shipper identification code. This code identifies the manufacture/shipper of the merchandise by a constructed code. The method for deriving the code can be found in Appendix 2. For the purposes of thiscode, the manufacturer should be construed to refer to the invoicing party or parties (manufacturers or other direct suppliers). The name and address of the invoicing party, whose invoice accompanies the CBP entry, should be used to construct the MID. Themanufacturer/shipper identification code is required for all entry summaries andentry/entry summaries, including informal entries, filed on the CBP Form 7501. For textile shipments, the manufacturer should be construed to refer to the actualmanufacturer in accordance with 19 CFR 102.23(a) and the Appendix to 19 CFR Part102 – Textile and Apparel Manufacturer Identification. Non-textile shipments may use the invoicing party or parties (manufacturers or otherdirect suppliers). When an entry summary covers merchandise from more than one manufacturer, recordthe word "MULTI" in this block. In column 28, indicate the MID Code corresponding toeach line item. If there is more than one vendor for a particular HTS number, separateline items will be required for each MID.
+    type: string
+    $linkedData:
+      term: manufacturerId
+      '@id': https://schema.org/identifier
+  exportingCountry:
+    title: Country of Origin
+    description: >-
+      Record the country of origin utilizing the International Organization for Standardization (ISO) country code located in Annex B of the HTS. The country of exportation is the country of which the merchandise was last part of thecommerce and from which the merchandise was shipped to the U.S. withoutcontingency of diversion.When an entry summary covers merchandise from more than one country of export,record the word "MULTI" in this block. In column 27, directly below the line number,prefixed with the letter "E," indicate the ISO code corresponding to each line item.Where line items are segregated by invoice, indicate the ISO code corresponding toeach invoice.For merchandise entering the customs territory from a U.S. FTZ, if multiple countries ofexport are involved, indicate the country of export with the greatest value. In the casewhere the merchandise is U.S. goods returned and the filer opts to pay the duty andreport only the Chapter 1 - 97 HTS number, report the country of invoice or exportation. Country of export U.S. will not be acceptable when country of origin is U.S.
```

Comment:
I'm not sure "contingency of diversion" is correct; might should be "contingency or diversion"
```suggestion
      Record the country of origin utilizing the International Organization for Standardization (ISO) country code located in Annex B of the HTS. The country of exportation is the country of which the merchandise was last part of the commerce and from which the merchandise was shipped to the U.S. without contingency of diversion. When an entry summary covers merchandise from more than one country of export, record the word "MULTI" in this block. In column 27, directly below the line number, prefixed with the letter "E," indicate the ISO code corresponding to each line item. Where line items are segregated by invoice, indicate the ISO code corresponding to each invoice. For merchandise entering the customs territory from a U.S. FTZ, if multiple countries of export are involved, indicate the country of export with the greatest value. In the case where the merchandise is U.S. goods returned and the filer opts to pay the duty and report only the Chapter 1 - 97 HTS number, report the country of invoice or exportation. Country of export U.S. will not be acceptable when country of origin is U.S.
```